.travis.yml

```diff
@@ -7,20 +7,16 @@ env:
         - customize=""
     matrix:
         - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
-          testflags="--test-type integ_tier1 --pytest-args='-x'
-              --copr networkmanager/NetworkManager-1.22-git"
+          testflags="--test-type integ_tier1 --pytest-args='-x'"
         - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
-          testflags="--test-type integ_tier1 --pytest-args='-x'
-              --copr networkmanager/NetworkManager-1.22-git"
+          testflags="--test-type integ_tier1 --pytest-args='-x'"
           customize="dnf remove -y openvswitch2.11 python3-openvswitch2.11;
             dnf install -y openvswitch2.13 python3-openvswitch2.13;
             systemctl restart openvswitch"
         - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
-          testflags="--test-type integ_tier2 --pytest-args='-x'
-              --copr networkmanager/NetworkManager-1.22-git"
+          testflags="--test-type integ_tier2 --pytest-args='-x'"
         - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
-          testflags="--test-type integ_slow --pytest-args='-x'
-              --copr networkmanager/NetworkManager-1.22-git"
+          testflags="--test-type integ_slow --pytest-args='-x'"
```

Comment:
If I remember correctly, we wanted to test against 1.22-git in order to identify issues earlier. Right now, we are testing against master too, do you think master and 1.26 stable is enough? @cathay4t

Reply:
For me. It's enough.
For NM bugs, let's leave it to their QE and CI.

Reply:
I agree with you. Thanks!